packages/clarity-js/src/data/baseline.ts

```diff
@@ -8,14 +8,14 @@ let buffer: BaselineData = null;
 let update: boolean = false;
 
 export function start(): void {
+    update = false;
     reset();
 }
 
 export function reset(): void {
     // Baseline state holds the previous values - if it is updated in the current payload, reset the state to current value after sending the previous state
     state = update ? { time: time(), event: Event.Baseline, data: buffer } : state;
-    buffer = buffer ? buffer : { visible: BooleanFlag.True, docWidth: 0, docHeight: 0, screenWidth: 0, screenHeight: 0 };
-    update = false;
```

Comment:
Removing update here will mean that we are sending baseline every time. Is that a desired thing? Earlier we were only sending when there was an update.